### ISSUE TITLE
@react-keycloak/nextjs throws window is not defined reference error after upgrading from 2.0.0-beta.6 to 2.0.0-beta.7

### ISSUE THREAD

**Issue by tiaanduplessis** [panz3r/react-keycloak#82]
**Describe the bug**

Firstly, thank you for maintaining this project. After upgrading a project from  `2.0.0-beta.6` of `@react-keycloak/nextjs` to `2.0.0-beta.7` I've started to get the following error when running Next.js in development:

```
ReferenceError: window is not defined
```

This is very strange as `2.0.0-beta.7` was only a version bump from what I can see https://github.com/panz3r/react-keycloak/blob/master/packages/nextjs/CHANGELOG.md#200-beta7-2020-07-20

**To Reproduce**

The issue has been reproduced here: https://github.com/tiaanduplessis/react-keycloak-next-ssr-issue

```
    "dependencies": {
        "@react-keycloak/nextjs": "2.0.0-beta.7",
        "cookie": "0.4.1",
        "keycloak-js": "10.0.2",
        "next": "9.4.4",
        "react": "16.13.1",
        "react-dom": "16.13.1"
    }
```

1. Set version to  `2.0.0-beta.7 ` in `package.json`
2. Run `npm install`
3. Run `npm run dev`

**Expected behavior**

Run without `window is not defined` error. 

**Desktop (please complete the following information):**

- OS: Ubuntu 20.04
- Browser: Brave
- Version 1.11.97 Chromium: 84.0.4147.89 (Official Build) (64-bit)

**Comment by panz3r** [comment]
Hi @tiaanduplessis ,

Thanks for your detailed feedback, I was able to reproduce the issue and track down its cause.
It turned out to be an issue introduced by an upgrade to the `rollup` configurations that was extracting the conditional `require` used by the SSR modules thus causing the reported `ReferenceError: window is not defined`.

I fixed the configuration and published a new versions of the 3 SSR modules (`nextjs`, `razzle` and `ssr`).

P.S. I'd suggest you to migrate your project to the new unified `@react-keycloak/ssr` module which is easier to integrate with future version of `nextjs` also I plan to deprecate the `@react-keycloak/nextjs` module soon.
